src/gregorio-utils.c

```diff
@@ -161,52 +161,62 @@ static char *get_output_filename(char *fbasename, const char *extension)
+%s\n\
 \n\
 This program is free software: you can redistribute it and/or modify\n\
 it under the terms of the GNU General Public License as published by\n\
 the Free Software Foundation, either version 3 of the License, or\n\
 (at your option) any later version.\n\
-\n");
+\n", copyright);
     printf("This program is distributed in the hope that it will be useful,\n\
 but WITHOUT ANY WARRANTY; without even the implied warranty of\n\
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n\
 GNU General Public License for more details.\n\
 \n\
 You should have received a copy of the GNU General Public License\n\
-along with this program.  If not, see <http://www.gnu.org/licenses/>.\n\n");
+along with this program.  If not, see <http://www.gnu.org/licenses/>.\n\
+");
 }
 
 static void print_usage(char *name)
 {
-    printf(_("\nUsage :\n%s [OPTION] {file}\n  where OPTION is :\n\
-\t-o file    writes output to specified file\n\
-\t-S         writes output to stdout\n\
-\t-F format  specifies output file format (default: gtex)\n\
-\t-l file    writes messages output to specified file (default: stderr)\n\
-\t-f format  specifies input file format (default: gabc)\n\
-\t-s         reads input from stdin\n\
-\t-p         generate point-and-click information\n\
-\t-h         displays this message\n\
-\t-V         displays %s version\n"), name, name);
-    printf(_("\t-L         displays licence\n\
-\t-v         verbose mode\n\
-\t-W         displays all warnings\n\
+    printf(_("Usage: %s [OPTION]... [-s | INPUT_FILE]\n\
+\nEngrave Gregorian chant scores, convert a gabc file to GregorioTeX.\n\n\
+Options:\n\
+  -o, --output-file FILE    write output to FILE,\n\
+                            default is basename(INPUT_FILE).FORMAT\n\
+  -S, --stdout              write output to stdout\n\
+  -s, --stdin               read input from stdin\n\
+  -l, --messages-file FILE  output messages to FILE (default: stderr)\n\
+  -F, --output-format FORMAT\n\
+                            specify output format (default: gtex)\n"), name);
+printf(_("  -f, --input-format FORMAT\n\
+                            specify input format (default: gabc)\n\
+  -p, --point-and-click     generate Lilypond point and click information\n\
+  -h, --help                print this help message\n\
+  -V, --version             print version and exit\n"));
+    printf(_("\
+  -L, --license             print licence\n\
+  -v, --verbose             verbose mode\n\
+  -W, --all-warnings        output warnings\n\
+  -d, --debug               output debug information\n\
+\n\
+Formats:\n\
+  gabc      gabc\n\
+  gtex      GregorioTeX\n\
+  dump      plain text dump (for debugging purpose)\n\
 \n\
-available formats are:\n\
-\t gabc      gabc\n\
-\t gtex      GregorioTeX\n\
-\t dump      simple text dump (for debugging purpose)\n\
-\n"));
+See <%s> for general documentation,\n\
+GregorioRef-%s.pdf and GregorioNabcRef-%s.pdf for full documentation.\
+\n"), PACKAGE_URL, VERSION, VERSION);
```

Comment:
Since these are `#define`, they can be inserted directly: `"...GregorioRef-" VERSION ".pdf..."`